addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -2286,162 +2121,146 @@ public Action event_CPCaptureStart(Handle hEvent, const char[] strName, bool bHi
-					int iHealth = GetClientHealth(i);
-					int iMaxHealth = SDK_GetMaxHealth(i);
-					if (iHealth < iMaxHealth || g_flTankLifetime[i] < GetGameTime() - 15.0)
+					//Tank super health handler
+					int iHealth = GetClientHealth(iClient);
+					int iMaxHealth = SDK_GetMaxHealth(iClient);
+					if (iHealth < iMaxHealth || g_flTankLifetime[iClient] < GetGameTime() - 15.0)
 					{
-						if (iHealth - g_iSuperHealthSubtract[i] > 0)
-							SetEntityHealth(i, iHealth - g_iSuperHealthSubtract[i]);
+						if (iHealth - g_iSuperHealthSubtract[iClient] > 0)
+							SetEntityHealth(iClient, iHealth - g_iSuperHealthSubtract[iClient]);
 						else
-							ForcePlayerSuicide(i);
+							ForcePlayerSuicide(iClient);
 					}
 					
-					// screen shake if tank is close by
+					//Screen shake if tank is close by
 					float flPosClient[3];
 					float flPosTank[3];
 					float flDistance;
-					GetClientEyePosition(i, flPosTank);
+					GetClientEyePosition(iClient, flPosTank);
 
-					for (int z = 1; z <= MaxClients; z++)
+					for (int i = 1; i <= MaxClients; i++)
 					{
-						if (IsClientInGame(z) && IsPlayerAlive(z) && IsSurvivor(z))
+						if (IsClientInGame(i) && IsPlayerAlive(i) && IsSurvivor(i))
 						{
-							GetClientEyePosition(z, flPosClient);
+							GetClientEyePosition(i, flPosClient);
 							flDistance = GetVectorDistance(flPosTank, flPosClient);
 							flDistance /= 20.0;
 							if (flDistance <= 50.0)
 							{
-								Shake(z, fMin(50.0 - flDistance, 5.0), 1.2);
+								Shake(i, fMin(50.0 - flDistance, 5.0), 1.2);
 							}
 						}
 					}
 				}
 
-				// kingpin
-				if (g_iSpecialInfected[i] == INFECTED_KINGPIN)
+				//Kingpin
+				if (g_nInfected[iClient] == Infected_Kingpin)
 				{
-					TF2_AddCondition(i, TFCond_TeleportedGlow, 1.5);
-
+					TF2_AddCondition(iClient, TFCond_TeleportedGlow, 1.5);
+					
 					float flPosClient[3];
 					float flPosScreamer[3];
 					float flDistance;
-					GetClientEyePosition(i, flPosScreamer);
-
-					for (int z = 1; z <= MaxClients; z++)
+					GetClientEyePosition(iClient, flPosScreamer);
+					
+					for (int i = 1; i <= MaxClients; i++)
 					{
-						if (IsValidLivingZombie(z))
+						if (IsValidLivingZombie(i))
 						{
-							GetClientEyePosition(z, flPosClient);
+							GetClientEyePosition(i, flPosClient);
 							flDistance = GetVectorDistance(flPosScreamer, flPosClient);
 							if (flDistance <= 600.0)
 							{
-								TF2_AddCondition(z, TFCond_TeleportedGlow, 1.5);
-								zf_screamerNearby[z] = true;
+								TF2_AddCondition(i, TFCond_TeleportedGlow, 1.5);
+								g_iScreamerNearby[i] = true;
 							}
-
 							else
 							{
-								zf_screamerNearby[z] = false;
+								g_iScreamerNearby[i] = false;
 							}
 						}
 					}
 				}
-
-				// stalker
-				if (g_iSpecialInfected[i] == INFECTED_STALKER)
+				
+				//Stalker
+				if (g_nInfected[iClient] == Infected_Stalker)
 				{
 					float flPosClient[3];
 					float flPosPredator[3];
```

Comment:
`flPosClient` should be `vecPosClient`, `flPosPredator` should be `vecPosPredator`